src/components/VtkObliqueView.vue

```diff
@@ -0,0 +1,671 @@
+      if (curImageID.value !== null) {
+        viewSliceStore.updateConfig(viewID.value, curImageID.value, {
+          slice,
+        });
+      }
+    };
+
+    // --- view proxy setup --- //
+
+    const { viewProxy, setContainer: setViewProxyContainer } =
+      useViewProxy<vtkLPSView2DProxy>(viewID, ViewProxyType.Oblique);
+
+    const resliceCursorRef = inject(VTKResliceCursor);
+    if (!resliceCursorRef) {
+      throw Error('Cannot access global ResliceCursor instance.');
+    }
+
+    const VTKViewType = computed(() => {
+      const viewAxisValue = viewAxis?.value;
+      switch(viewAxisValue) {
+        case 'Coronal': return ViewTypes.XZ_PLANE;
+        case 'Sagittal': return ViewTypes.YZ_PLANE;
+        default:
+          return ViewTypes.XY_PLANE;
+      }
+    });
+
+    const { baseImageRep } = useSceneBuilder<
+      vtkObliqueRepresentationProxy
+    >(viewID, {
+      baseImage: curImageID
+    });
+
+    const obliqueRep = baseImageRep?.value;
+    if (obliqueRep) {
+      switch(viewDirection.value) {
+        case 'Left':
+        case 'Right':
+          obliqueRep.setOutlineColor([1, 0, 0]);
+          break;
+        case 'Posterior':
+        case 'Anterior':
+          obliqueRep.setOutlineColor([0, 1, 0]);
+          break;
+        case 'Superior':
+        case 'Inferior':
+          obliqueRep.setOutlineColor([0, 0, 1]);
+          break;
+        default:
+          obliqueRep.setOutlineColor([0, 0, 0]);
+          break;
+      };
+    }
+
+    onBeforeMount(() => {
+      // do this before mount, as the ManipulatorTools run onMounted
+      // before this component does.
+      viewProxy.value.getInteractorStyle2D().removeAllManipulators();
+    });
+
+    const updateViewFromResliceCursor = () => {
+      const rep = baseImageRep?.value;
+      const resliceCursor = resliceCursorRef?.value;
+      const state = resliceCursor?.getWidgetState() as ResliceCursorWidgetState;
+      if (resliceCursor && rep) {
+        const planeOrigin = state.getCenter();
+        const planeNormal = resliceCursorRef.value.getPlaneNormalFromViewType(VTKViewType.value);
+        rep.getSlicePlane().setNormal(planeNormal);
+        rep.getSlicePlane().setOrigin(planeOrigin);
+        if (curImageData.value) {
+          resliceCursorRef.value.updateCameraPoints( viewProxy.value.getRenderer(), VTKViewType.value, false, false, true);
+        }
+      }
+    }
+
+    onMounted(() => {
+      setViewProxyContainer(vtkContainerRef.value);
+      viewProxy.value.setOrientationAxesVisibility(false);
+
+      // Initialize camera points during construction
+      if (curImageData.value) {
+        resliceCursorRef.value.updateCameraPoints(viewProxy.value.getRenderer(), VTKViewType.value, true, false, true);
+      }
+
+      const onPlanesUpdated = useVTKCallback(
+        resliceCursorRef.value.getWidgetState().onModified
+      );
+
+      onPlanesUpdated(() => {
+        updateViewFromResliceCursor();
+      });
```

Comment:
This should be moved outside of onMounted, so that the lifecycle methods in useVTKCallback work.

```
      const onPlanesUpdated = useVTKCallback(
        computed(() => resliceCursorRef.value?.getWidgetState().onModified)
      );
```